### PR TITLE
Portfolio: add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by amon-cofie**
This is my pull request to add a contact form to my portfolio project. I did the following things in it. 

- Added markup for the "contact me" section
- Added styles for the "contact me" section 
- Installed and checked for linter errors.
- Fixed all linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -319,6 +319,50 @@ <h3>LET'S CONNECT</h3>
...
+        <div class="contact-me-background">
+          <article class="contact-me">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </article>
+          <form action="#" method="post">
```
Comment (ShahierNashaat) [inline index.html:332]: - [ ] Kindly be noticed that the `action` value should be similar to `https://formspree.io/f/{form_id}`.

File: index.html [hunk 2]
```diff
@@ -319,6 +319,50 @@ <h3>LET'S CONNECT</h3>
...
+            <ul>
+              <li>
+                <input
+                  type="text"
+                  name="user_name"
+                  id="name"
+                  placeholder="Enter your name here"
+                />
```
Comment (ShahierNashaat) [inline index.html:340]: - [ ] Kindly be noticed that the `name` input field should have the `required` and `maxlength="30"` attributes.
- [ ] Also you should make sure that your design is like the Figma Design.

Figma Design | Your Design
------------------ | -----------------
Screenshot from 2022-08-23 17-32-22 | Screenshot from 2022-08-23 17-29-22

File: index.html [hunk 3]
```diff
@@ -319,6 +319,50 @@ <h3>LET'S CONNECT</h3>
...
+              </li>
+              <li>
+                <input
+                  type="email"
+                  name="user_email"
+                  id="email"
+                  placeholder="Enter your email here"
+                />
```
Comment (ShahierNashaat) [inline index.html:348]: - [ ] Kindly be noticed that the `email` input field should have the `required` attribute.

File: index.html [hunk 4]
```diff
@@ -319,6 +319,50 @@ <h3>LET'S CONNECT</h3>
...
+              </li>
+              <li>
+                <textarea
+                  name="user_msg"
+                  id="msg"
+                  cols="30"
+                  rows="10"
+                  placeholder="Write your message here"
+                ></textarea>
```
Comment (ShahierNashaat) [inline index.html:357]: - [ ] Kindly be noticed that the `textarea` tag should have the `required` and `maxlength="500"` attributes.

File: index.html (hunk 1 above)
Comment (amon-cofie) [reply index.html:332]: Thanks a lot @ShahierNashaat

File: index.html (hunk 2 above)
Comment (amon-cofie) [reply index.html:340]: Will do

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @amon-cofie   , 

Great work on making the changes requested by a previous reviewer 👏
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README
✔️ Good commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**